### PR TITLE
Bootstrapped Django 5.1.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: django/db/migrations/autodetector.py [hunk 1]
```diff
@@ -608,12 +606,10 @@ def generate_created_models(self):
                     if getattr(field.remote_field, "through", None):
                         related_fields[field_name] = field
 
-            # Are there indexes/unique|index_together to defer?
+            # Are there indexes/unique to defer?
```
Comment (ngnpope) [inline django/db/migrations/autodetector.py:609]:
```suggestion
            # Are there indexes/unique_together to defer?
```

**Comment by felixxm** [comment]
buildbot

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by nessita** [comment]
It seems that there are some `versionadded` that may need removing?

```
docs/topics/auth/customizing.txt:734:        .. versionadded:: 4.1.8
docs/ref/settings.txt:1070:.. versionadded:: 3.2.18
```

**Review by carltongibson — APPROVED** [review]
(Bar @nessita's query) I think this looks good. 👍 Thanks!

**Comment by felixxm** [comment]
> It seems that there are some `versionadded` that may need removing?
> 
> ```
> docs/topics/auth/customizing.txt:734:        .. versionadded:: 4.1.8
> docs/ref/settings.txt:1070:.. versionadded:: 3.2.18
> ```

I removed remaining `versionadded` annotations.

**Comment by felixxm** [comment]
Thanks for reviews :+1:

**Pull request merged**